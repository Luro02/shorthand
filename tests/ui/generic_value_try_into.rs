use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand(enable(try_into))]
struct DoNotDoThis<VALUE> {
    value: VALUE,
}

fn main() {}
