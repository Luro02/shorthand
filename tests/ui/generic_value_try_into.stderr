error: a generic called `VALUE` is not supported, please rename it.
 --> $DIR/generic_value_try_into.rs:5:19
  |
5 | struct DoNotDoThis<VALUE> {
  |                   ^^^^^^^
